en/mulesoft/modules/ROOT/pages/ms_composer_invocable_flows.adoc

```diff
@@ -0,0 +1,263 @@
+= Invoke a Composer Flow 
+
+Invoke a Composer flow from external systems to enable deeper interoperability and integrated capabilities across Salesforce. You can also authenticate every invocable flow for a secure and protected automation experience, create API-compatible invocable actions for seamless integration with diverse systems and applications, and build powerful automations by leveraging the built-in support for input fields.
+
+When your invocable flow is activated, Composer exposes a unique flow URL and a unique pair of client credentials (client ID and client secret) so that your flow can be invoked from external systems. You can also reset these credentials for security best practices. Invocable flows are invoked by using an HTTP call to the flow URL and by using the flow client credentials as authentication. 
+
+To invoke a Composer flow, you must:
+
+. <<create-an-invocable-composer-flow, Create an invocable Composer flow.>> 
+. <<invoke-the-composer-flow-from-an-external-system, Invoke the Composer flow from an external system.>>
+
+[[create-an-invocable-composer-flow]]
+== Create an Invocable Composer Flow
+
+Create an invocable Composer flow so you can later invoke it from an external system. To create your invocable Composer flow, you must:
+
+. <<configure-your-invocable-composer-flow, Configure your invocable Composer flow.>>
+. <<test-your-invocable-composer-flow, Test your invocable Composer flow.>>
+. <<activate-your-invocable-composer-flow, Activate your invocable Composer flow.>>
+
+[[configure-your-invocable-composer-flow]]
+=== Configure Your Invocable Composer Flow
+
+To configure an invocable Composer flow:
+
+. Access Composer.
+. xref:ms_composer_flows.adoc#create-a-new-flow[Create a new flow from scratch].
+. In the *What Should Start This Flow?* dialog, in the *Interaction* section, click *Invoke* to create an invocable flow. + 
+The *Trigger: Invoke this flow from an external system via HTTP POST Request* trigger appears. 
```

Comment:
This text does not make sense as is.

Reply:
I know - that is sadly what appears in the UI.